build.gradle.kts

```diff
@@ -3,7 +3,7 @@
  */
 
 plugins {
-    kotlin("jvm") version "1.7.10"
+    kotlin("jvm")
```

Comment:
Not related to explicit api mode, was passing by and noticed a potentially confusing duplication of used Kotlin version

Resolution happens in [settings.gradle](https://github.com/Kotlin/kotlinx.reflect.lite/blob/23297f144bc2eda7d2ca6ae0aeadaa17c23f29e4/settings.gradle.kts#L8-L15) based on `kotlin_version` gradle property

Reply:
Good catch